src/01-highlights-list/index.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * Highlights List
+ * A stylised list displaying bullet points and a title
+ */
+import { __ } from '@wordpress/i18n';
+import { registerBlockType, registerBlockStyle } from '@wordpress/blocks';
+import { RichText } from '@wordpress/block-editor';
+
+registerBlockType('mojblocks/highlights-list', {
+    title: __('Highlights List', 'mojblocks'),
+    icon: 'list-view',
+    category: 'mojblocks',
+    example: {
+        attributes: {
+            listTitle: 'This is a highlights list title',
+            listText: 'This is a list item',
+        },
+    },
+    attributes: {
+        listTitle: {
+            type: 'string',
+            source: 'html',
+            selector: '.mojuk-highlights-list__heading-text'
+        },
+        listText: {
+            type: 'array',
+            source: 'children',
+            multiline: 'li',
+            selector: '.mojuk-highlights-list__content > ul'
+        }
+    },
+
+    edit: props => {
+        let {
+            attributes: {
+                listTitle,
+                listText
+            },
+            className,
+            setAttributes
+        } = props;
+
+        // Grab newListTitle, set the value of listTitle to newListTitle.
+        let onChangeListTitle = newListTitle => {
+            setAttributes({ listTitle: newListTitle });
+        };
+
+        // Grab newListText, set the value of listText to newListText.
+        let onChangeListText = newListText => {
+            setAttributes({ listText: newListText });
+        };
+
+        return (
+            <div className={`${className}`}>
+                <div className="mojuk-highlights-list__heading-container">
+                    <h3 className="mojuk-highlights-list__heading">
+				  <span role="text">
+					<span className="mojuk-highlights-list__heading-text">
+					  <RichText
+                          placeholder={__('Highlights title', 'mojblocks')}
+                          value={listTitle}
+                          onChange={onChangeListTitle}
+                      />
+					</span>
+				  </span>
+                    </h3>
+                </div>
+                <div className={'mojuk-highlights-list__content'}>
+                    <RichText
+                        tagName='ul'
+                        multiline='li'
+                        placeholder={__('Highlights item', 'mojblocks')}
+                        onChange={onChangeListText}
+                        value={listText}
+                    />
+                </div>
+            </div>
+        );
+    },
+
+    save: props => {
+        let {
+            attributes: {
+                listTitle,
+                listText
+            }
+        } = props;
+
+        return (
+            <div className="mojuk-grid-column-width mojuk-highlights-list--type">
```

Comment:
@wilson1000-MoJ Just seen this mojuk-grid-column-width class. Just curious where is this defined in sass? Or is this just our own naming convention?